cmake/Tools/Platform/Android/android_deployment.py

```diff
@@ -94,7 +94,8 @@ def __init__(self, dev_root, build_dir, configuration, android_device_filter, cl
             if asset_mode == 'PAK':
                 self.local_asset_path = self.dev_root / 'Pak' / f'{game_name.lower()}_{asset_type}_paks'
             else:
-                self.local_asset_path = self.dev_root / game_name / 'Cache' / asset_type
+                # Assets layout folder when assets are not included into APK is 'app/src/assets'
+                self.local_asset_path = self.build_dir / 'app/src/assets'
```

Comment:
If `asset_mode` is `loose` and the assets need to go into the apk, shouldn't this be 'app/src/main/assets'? Or we could just point to the cache folder directly like it's currently set. Why do we need to use the path to the copy?

Reply:
Shouldn't we also check that asset_mode == "LOOSE" here as well

Reply:
>Shouldn't we also check that asset_mode == "LOOSE" here as well

Not necessary. Not PAK is good because the other 2 asset modes (LOOSE or VFS) have the same location of assets layout.